pymatgen/io/cif.py

```diff
@@ -314,18 +314,18 @@ class CifParser:
     def __init__(
         self,
         filename: PathLike | StringIO,
-        occupancy_tolerance: float = 1.0,
+        occupancy_tolerance: float = 0.0,
```

Comment:
I think original design here makes more sense; occupancy_tolerance defaults to 1.0, which is the expected highest occupancy of an atom site that could go. By default, when occupancy is > 1, throws an error.
Waiting on someone more experienced to comment on cases where `check_occu = False`

Reply:
Yes I think this is not a core concern here, and I have reverted this in 188b2d6ab6328ec2fd8e131d845c200f019e43d7